File: boot-cache/src/main/scala/com/hjf/boot/demo/boot/cache/scala/ManService.scala
package com.hjf.boot.demo.boot.cache.scala

import org.springframework.stereotype.Service

/**
  * name: com.hjf.boot.demo.boot.cache.scala
  * author: mickjoust
  * date: 2018/5/10
  **/
trait ManService {

  def findByName(name: String): Man

}

File: boot-other/src/main/kotlin/com/hjf/boot/demo/boot/cache/kotlin/ManService.kt
package com.hjf.boot.demo.boot.cache.kotlin

//import org.springframework.data.jpa.repository.JpaRepository

/**
 * name: com.hjf.boot.demo.boot.high_concurrency.kotlin
 * author: mickjoust
 * date: 2018/5/9
 **/
interface ManService  {
    fun findByName(name: String): Man
}